arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -0,0 +1,2508 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Service Map",
+    "description": "Service Map API Reference",
+    "version": "2015-11-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/features/serviceMap/machines": {
+      "get": {
+        "tags": [
+          "Machines"
+        ],
+        "operationId": "Machines_List",
```

Comment:
This operation retrieves machines in a workspace and there can be multiple workspaces in a resource group. All operations here operate in the context of a workspace, so a name like Machines_ListByWorkspace is redundant and this is why the operationId is Machines_List.